src/riak_ql_parser.yrl

```diff
@@ -135,17 +148,36 @@ Statement -> StatementWithoutSemicolon semicolon : '$1'.
 
 GroupBy -> group by Fields: {group_by, '$3'}.
 
+SortDirection -> asc: '$1'.
```

Comment:
Rewrote the rules using the official idioms. I have to note that there is no rule for the LIMIT [OFFSET] clause, oddly enough, which I had to invent ad hoc.